src/tasks.py

```diff
@@ -45,6 +47,28 @@ def ami_for_project(pname):
     # good for figuring out filters 
     #print results[0].__dict__
 
+@task
```

Comment:
mutual dependencies: keeping it in `aws` means `aws` has to import `bakery`, which triggered a circular dependency